src/content-helper/editor-sidebar/smart-linking/component.tsx

```diff
@@ -483,7 +487,6 @@ export const SmartLinkingPanel = ( {
 				e.message = snackBarMessage;
 			}
 
-			console.error( e ); // eslint-disable-line no-console
```

Comment:
I found this `console.error( e )` to be quite noisy because it often outputs the same message 3 times due to retries. In addition, we're now showing the actual error message in the snackbar, so I think we're OK with removing this.

Having said that, I'm OK with bringing this back if we think it's better to have.

Reply:
Perhaps we should also remove the one here:

https://github.com/Parsely/wp-parsely/blob/f40cd5fbfacf83aa7549b840a9fbc28d69485dfa/src/content-helper/editor-sidebar/smart-linking/component.tsx#L541-L545

I actually think that `console.error` is actually the one throwing the same errors on each retry. In theory, this one that you removed should only be called if there's an error after all the retries attempts. 

I will leave to your discretion if we want to remove both, or keep this one and remove the other instance.

Reply:
I'm actually restoring the `console.error()`, it will make things simpler for now. If we get noise again, we can look into it.